CMakeLists.txt

```diff
@@ -26,58 +26,99 @@
 #
 
 # yue.nicholas@gmail.com
-CMAKE_MINIMUM_REQUIRED(VERSION 3.1)
+
+CMAKE_MINIMUM_REQUIRED ( VERSION 3.1 )
+
+SET ( CMAKE_CXX_STANDARD 11 )
+SET ( CMAKE_DISABLE_SOURCE_CHANGES ON )
+SET ( CMAKE_DISABLE_IN_SOURCE_BUILD ON )
 
 PROJECT ( OpenVDB )
 
-ENABLE_TESTING()
+ENABLE_TESTING ()
 
-find_program(CCACHE_FOUND ccache)
-if(CCACHE_FOUND)
-    set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ccache)
-    set_property(GLOBAL PROPERTY RULE_LAUNCH_LINK ccache)
-endif(CCACHE_FOUND)
+FIND_PROGRAM ( CCACHE_FOUND ccache )
+IF ( CCACHE_FOUND )
+    SET_PROPERTY ( GLOBAL PROPERTY RULE_LAUNCH_COMPILE ccache )
+    SET_PROPERTY ( GLOBAL PROPERTY RULE_LAUNCH_LINK ccache )
+ENDIF ( CCACHE_FOUND )
 
 OPTION ( OPENVDB_BUILD_UNITTESTS "Build the OpenVDB unit tests" ON )
 OPTION ( OPENVDB_BUILD_DOCS "Build the OpenVDB documentation" OFF )
-OPTION ( OPENVDB_BUILD_PYTHON_MODULE "Build the pyopenvdb Python module" ON )
+OPTION ( OPENVDB_BUILD_PYTHON_MODULE "Build the pyopenvdb Python module" OFF )
 OPTION ( OPENVDB_BUILD_HOUDINI_SOPS "Build the Houdini SOPs" OFF )
 OPTION ( OPENVDB_BUILD_MAYA_PLUGIN "Build the Maya plugin" OFF )
 OPTION ( OPENVDB_BUILD_CORE "Build the OpenVDB core" ON )
 OPTION ( OPENVDB_ENABLE_RPATH "Build with RPATH information" ON )
-OPTION ( OPENVDB_ENABLE_3_ABI_COMPATIBLE "Build with OpenVDB v3 ABI" ON )
 OPTION ( OPENVDB_DISABLE_BOOST_IMPLICIT_LINKING "Disable the implicit linking of Boost libraries on Windows" ON )
 OPTION ( OPENVDB_CXX_STRICT "Enable or disable a set of pre-defined compiler warnings for clang and gcc" OFF )
+OPTION ( OPENVDB_ENABLE_3_ABI_COMPATIBLE "Build with OpenVDB ABI 3 (deprecated - see "
+  "OPENVDB_ABI_VERSION_NUMBER)" OFF )
 
-SET ( OPENVDB_ABI_VERSION_NUMBER "" CACHE STRING "build for compatibility
-with version N of the OpenVDB Grid ABI, where N is 2, 3, 4, etc.
-(some newer features will be disabled)")
+SET ( OPENVDB_ABI_VERSION_NUMBER "" CACHE STRING "Build for compatibility with version N of the
+    OpenVDB Grid ABI, where N is 3, 4, 5 etc. (some newer features will be disabled)")
 
-IF (OPENVDB_BUILD_HOUDINI_SOPS)
+IF ( OPENVDB_BUILD_HOUDINI_SOPS )
+  # @todo why?
```

Comment:
Possibly because, prior to the introduction of HBoost, we built against Houdini's version of Boost, and Houdini didn't ship with Boost.Python.

Reply:
Ah makes sense, thanks - added as a comment